wetmap/src/components/reusables/label/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useId } from 'react';
```

Comment:
what is this "useId"? it's throwing and error that its not from react? Is it imported form something else?

Reply:
It's a standard React Hook for generating unique string ids: https://react.dev/reference/react/useId
It was throwing and error because this hook was introduced in React 18 but we used "@types/react": "^17.x.x".
I've updated "@types/react" to version 18 which fixed the problem. 
Please also check on your side.